packages/server/package.json

```diff
@@ -79,7 +79,7 @@
     "dotenv": "8.2.0",
     "form-data": "4.0.0",
     "global-agent": "3.0.0",
-    "google-spreadsheet": "4.1.2",
+    "google-spreadsheet": "npm:@budibase/google-spreadsheet@4.1.2",
```

Comment:
As this is temporal, I use a package override. This will help us as we will not need to update code references

Reply:
very nice - good way to solve this!